playbulb-candle/sw.js

```diff
@@ -1,27 +1,40 @@
-self.addEventListener('fetch', function(event) {
-  event.respondWith(
-    caches.open('playbulb-candle').then(cache => {
-      return cache.match(event.request).then(response => {
+const addToCache = (request, networkResponse) => {
```

Comment:
Kinda. `function` is clearer that we're defining a function, it takes fewer tokens, and it's more familiar to people reading the code. `const` might be clearer that we don't overwrite the function? If there's another reason to pick `const` that's fine, but I don't think we should switch to it against the three reasons to prefer `function`.